charts/monitoring-config/dashboards/graphql-stats.json

```diff
@@ -284,7 +315,7 @@
           "refId": "A"
         }
       ],
-      "title": "Content Store response time (6H)",
+      "title": "Content store response time (6h)",
```

Comment:
Reviewers please ignore this, I accidentally deleted this title and obviously didn't put it back properly. Seeing as this panel is removed in the next commit anyway this doesn't seem worth correcting.